Samples/SwiftKitSampleApp/src/main/java/com/example/swift/HelloJava2Swift.java

```diff
@@ -95,6 +91,12 @@ static void examples() {
             });
         }
 
+        try (var arena = AllocatingSwiftArena.ofConfined()) {
+            var bytes = arena.allocateFrom("hello");
+            var dat = Data.init(bytes, bytes.byteSize(), arena);
+            MySwiftLibrary.globalReceiveSomeDataProtocol(dat);
```

Comment:
@ktoso I'd be happy to make this and `Data` thing test cases, but where exactly I should write? Any PR testing runs `Samples:SwiftKitSampleApp:test`?

Reply:
Done!